### ISSUE TITLE
After install, "CoCo format" item colors all default to grey

### ISSUE THREAD

**Issue by BlueNinjaSoftware** [GeorgeAlexandria/CoCo#4]
After installing the extension and restarting VS (2017), all of the "CoCo format" colors defined in Tools / Options / Environment / Fonts and Colors are set to "default". I'd suggest having them initially default to some set color scheme like in the example screen shots.

(An initial peek at the code didn't reveal any pre-set defaults, so I assume this isn't simply a bug, but by all means correct me if I'm wrong!)

**Comment by GeorgeAlexandria** [comment]
No, you are right it's not a bug. I think it will be very strange when you install the new extension and it applies immediately a some pre-set colors to a code. 

However, it's only my opinion, so if someone else agree with you maybe I'll set the default values

**Comment by BlueNinjaSoftware** [comment]
I can see your point. Maybe a "Use Defaults" option could be added to the custom dialog (Issue #3), as I don't know of a clean way to pop up a "Would you like to use default colors?" dialog upon extension install.

Perhaps make a note in the readme that the colors don't have defaults, so that users don't think the extension isn't working, in case they expected to see the screen shot results on their system right away? It looks like a good extension, and it would be a shame for people to uninstall it thinking it was broken, simply because they didn't know they had to set the colors.

Thanks!
